randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1455,21 +1470,29 @@ Extra - map_name: rm_a5b05
   > Tunnel to Spiky Spider Trial Reward Room
       Any of the following:
           All of the following:
-              Morph Ball and Mid-Air Morph (Advanced)
+              # Suitless Method
+              Morph Ball
+              Any of the following:
+                  Mid-Air Morph (Advanced)
+                  Power Grip and Movement (Intermediate)
               Any of the following:
```

Comment:
Please add a comment here. Something like "From crumble blocks to top" maybe? theres probably something better out there